README.md

```diff
@@ -4,12 +4,44 @@
 [![License](https://img.shields.io/cocoapods/l/Whisper.svg?style=flat)](http://cocoadocs.org/docsets/Whisper)
 [![Platform](https://img.shields.io/cocoapods/p/Whisper.svg?style=flat)](http://cocoadocs.org/docsets/Whisper)
 
+## Description
+
+Break the silence of your UI whispering or shouting at it. **Whisper** is a component that will make the task of display messages and in-app notifications simple. It has two different views inside.
```

Comment:
`Break the silence of your UI whispering or shouting at it.`
Shouldn't it be a comma in there somewhere? :sunglasses: